python/custreamz/custreamz/tests/test_dataframes.py

```diff
@@ -22,6 +23,18 @@
 cudf = pytest.importorskip("cudf")
 
 
+@pytest.fixture(scope="module")
+def disable_distributed_gc_diagnosis():
```

Comment:
Shouldn't this fixture have been applied to a specific test? Or have an `autouse=True`?

Reply:
Ahh good catch. I meant to apply it to the `client` fixture but forgot. I'll fix that now.